Solutions/LastPass/Data Connectors/LastPassAPIConnector.json

```diff
@@ -1,123 +1,132 @@
-                            "authType": "APIKey",
-                            "APIKeyName": "provhash",
-                            "IsAPIKeyInPostPayload": true
-                        },
-                        "request": {
-                            "apiEndpoint": "https://lastpass.com/enterpriseapi.php",
-                            "rateLimitQPS": 2,
-                            "httpMethod": "Post",
-                            "queryTimeFormat": "yyyy-MM-ddTHH:mm:ssZ",
-                            "retryCount": 3,
-                            "queryWindowInMin": 10,
-                            "timeoutInSeconds": 120,
-                            "queryParametersTemplate": "{'cid': '12537091', 'cmd': 'reporting', 'data': { 'from': '{_QueryWindowStartTime}', 'to': '{_QueryWindowEndTime}' }, '{_APIKeyName}': '{_APIKey}'}",
-                            "isPostPayloadJson": true
-                        },
-                        "response": {
-                            "eventsJsonPaths": [
-                                "$.data"
-                            ],
-                            "successStatusJsonPath": "$.status",
-                            "successStatusValue": "OK",
-                            "convertChildPropertiesToArray": true
-                        },
-                        "paging": {
-                            "pagingType": "NextPageToken",
-                            "nextPageParaName": "next",
-                            "nextPageTokenJsonPath": "$.next"
-                        }
-                    }
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "workspace": {
+      "type": "string",
+      "defaultValue": ""
+    }
+  },
+  "resources": [
+    {
+      "id": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',resourceGroup().name,'/providers/Microsoft.OperationalInsights/workspaces/',parameters('workspace'),'/providers/Microsoft.SecurityInsights/dataConnectors/',guid(subscription().subscriptionId))]",
+      "name": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',guid(subscription().subscriptionId))]",
+      "apiVersion": "2021-03-01-preview",
+      "type": "Microsoft.OperationalInsights/workspaces/providers/dataConnectors",
+      "kind": "APIPolling",
+      "properties": {
+        "connectorUiConfig": {
+          "title": "LastPass Enterprise - Reporting (test)",
+          "publisher": "The Collective Consulting BV",
+          "descriptionMarkdown": "The [LastPass Enterprise](https://www.lastpass.com/products/enterprise-password-management-and-sso) connector provides the capability to LastPass reporting (audit) logs into Azure Sentinel. The connector provides visibility into logins and activity within LastPass (such as reading and removing passwords).",
+          "graphQueriesTableName": "LastPass_BYOC_CL",
```

Comment:
@ThijsLecomte - Can you also change the "graphQueriesTableName" to something like "LastPassNativePoller_CL"?